masonite/facades/Auth.py

```diff
@@ -1,10 +1,12 @@
-''' Authentication Class '''
+"""Authentication Class"""
```

Comment:
Let's put a nice space there. so instead of `"""Authentication Class"""` it should be `""" Authentication Class """`